mate-panel/panel-menu-button.c

```diff
@@ -694,6 +694,8 @@ panel_menu_button_load (const char  *menu_path,
 		return;
 	}
 
+	gtk_widget_set_name(GTK_WIDGET (button),"mate-panel-main-menu-button");
```

Comment:
style nit: should have spaces before `(` and after `,`.
```suggestion
	gtk_widget_set_name (GTK_WIDGET (button), "mate-panel-main-menu-button");
```